ci/prepare_clean_env.sh

```diff
@@ -20,9 +20,10 @@ conda activate ./conda/${env_name}
 conda list
 printenv
 
-pip install cmake
-pip install -v -e .
-# pip install horovod --no-cache-dir -U
-python -m spacy download en
-python -m spacy download de
-python -m nltk.downloader all
+python3 -m pip install awscli
```

Comment:
We can install awscli through conda-forge `conda install -c conda-forge awscli`. This may fix the issue in CI.

Reply:
I realized that the issue happens on the batch job and it was the ci/batch/docker/gluon_nlp_job.sh I should be updating.

Reply:
We are using the CI docker in master branch here and the docker setting in v0.x branch are currently not used for v0.x CI. I keep the docker setting and scripts here for some future work that we create a job queue on batch for v0.x CI so that we can maintain the CI dockers seperately.